Index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="navContainer">
+                <div ><a href="#"><span class="nick">Elio Nel</span></a></div>
+                <div ><img src="./icons/IconMenu.png" alt="menu icon"></div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="sectionContainer">
+                <div class="upperCircle">
+                </div>
+                <div class="lowerCircle">
+                </div>
```

Comment:
@Deepakdanger The upperCircle and lowerCircle classes are already using background image, I used 2 separate divs because it allows the images to be aligned to the edges; so when we change the dimension of the preview the images are automatically accommodated to the edges. If I include only one image, it stays in a fixed position and does not fit the edges. I kindly ask you to validate this point.